website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -310,7 +310,11 @@ A `upgrade_settings` block supports the following:
 
 * `node_soak_duration_in_minutes` - (Optional) The amount of time in minutes to wait after draining a node and before reimaging and moving on to next node.
 
-* `max_surge` - (Required) The maximum number or percentage of nodes which will be added to the Node Pool size during an upgrade.
+* `max_surge` - (Optional) The maximum number or percentage of nodes which will be added to the Node Pool size during an upgrade. Exactly one of `max_surge` or `max_unavailable` must be specified.
+
+* `max_unavailable` - (Optional) The maximum number or percentage of nodes which can be unavailable during the upgrade. Exactly one of `max_surge` or `max_unavailable` must be specified.
```

Comment:
should we remove `Exactly one of `max_surge` or `max_unavailable` must be specified.` from the end of these and add it as a note below instead?